packages/mural-picker/src/components/card-list-item/styles.scss

```diff
@@ -89,8 +89,38 @@
     font-size: 0.675em;
     width: 140px;
 
+    .card-action-area {
+      padding: 0.8em;
+    }
+
+    .card-thumbnail {
+      height: 74px;
+      border-radius: 0.7em;
```

Comment:
Instead of having a different `border-radius` per card size, would be nice to have the same `border-radius` for all card sizes. Let's update the `border-radius` at line 28 for: `border-radius: 8px;`